src/irmin-pack/unix/chunked_suffix.ml

```diff
@@ -232,15 +254,42 @@ module Make (Io : Io.S) (Errs : Io_errors.S with module Io = Io) = struct
       Ao.open_ro ~path ~end_poff ~dead_header_size
     in
     let+ inventory = Inventory.open_ ~start_idx ~chunk_num ~open_chunk in
-    { inventory; root; dead_header_size }
+    { start_idx; inventory; root; dead_header_size }
 
+  let start_idx t = t.start_idx
```

Comment:
`start_idx` is something we can retrieve from the array of chunks, instead of adding it in `t`, no?